nepomuk/src/main/java/edu/asu/diging/gilesecosystem/nepomuk/core/files/impl/FileStorageManager.java

```diff
@@ -119,5 +125,43 @@ private boolean createDirectory(String dirPath) {
     public void setFileTypeFolder(String fileTypeFolder) {
         this.fileTypeFolder = fileTypeFolder;
     }
+    
+    public String getStoragePath(String username, String uploadId,
+            String documentId) {
+        return baseDirectory + File.separator 
+                + getFileFolderPathInBaseFolder(username, uploadId, documentId);
+    }
+    
+    public void deleteFile(String username, String uploadId, String documentId, String fileName) throws NepomukFileStorageException {
+        Path path = Paths.get(getStoragePath(username, uploadId, documentId) + File.separator + fileName);
```

Comment:
`getStoragePath(username, uploadId, documentId)` is used multiple times in this method, making it hard to easily see that it always refers to the same path. Just put it in a variable and use the variable instead.